### ISSUE TITLE
TypeError: Cannot read property 'forEach' of undefined

### ISSUE THREAD

**Issue by mwetter** [lbl-srg/modelica-json#107]
The command
```bash
git clone git@github.com:lbl-srg/modelica-buildings.git
cd modelica-buildings
git checkout 067b6e41640a5813fad108b61196402ff5d82da8
cd -

git clone git@github.com:lbl-srg/modelica-json.git
cd modelica-json/
make install
make compile
node app.js -f ../modelica-buildings/Buildings/Controls/OBC -o json -m modelica -d out
```
fails with
```
TypeError: Cannot read property 'forEach' of undefined
```

**Comment by mwetter** [comment]
@JayHuLBL : What is the status on this? This is important to fix soon as it will allow us to automate some testing/review of control sequences that are being developed by an increasing number of contributors.

**Comment by JayHuLBL** [comment]
@mwetter It will be addressed by PR #120.
